taichi/backends/vulkan/runtime.cpp

```diff
@@ -440,57 +447,88 @@ void VkRuntime::launch_kernel(KernelHandle handle, RuntimeContext *host_ctx) {
          /*export_sharing=*/false, AllocUsage::Storage});
   }
 
+  // Create context blitter
   auto ctx_blitter = HostDeviceContextBlitter::maybe_make(
       &ti_kernel->ti_kernel_attribs().ctx_attribs, host_ctx, device_,
       host_result_buffer_, ctx_buffer.get(), ctx_buffer_host.get());
 
   std::unordered_map<int, DeviceAllocation> ext_arrays;
+  std::unordered_map<int, size_t> ext_array_size;
```

Comment:
I guess `ext_` is no longer the accurate term, because it covers NDarray as well?